src/components/app.css

```diff
@@ -8,46 +8,144 @@
 }
 .northy {
     background-image: url('../assets/north.png');
-  background-size: cover;
-  font-size: 1.45em;
-  font-weight: bold;
-  display: flex;
-  text-align: center;
-  box-shadow: 2px 2px 16px 1px rgba(0, 0, 0, 0.3);
-    padding: 60px;
+   border-left: solid darkblue 2px;
 }
 
 .southy { 
-    background-image: url('../assets/south.png');
-  background-size: cover;
-  font-size: 1.3em;
-  font-weight: bold;
+  background-image: url('../assets/south.png');
+}
+
+.westy {
+  background-image: url('../assets/west.png');
+}
+
+.easty {
+background-image: url('../assets/east.png');
+}
+
+.capitalCard {
   display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  /* width: 150px;
+  height: 150px; */
+  padding: 1em;
+}
+
+.capitalCard img {
+  width: 100%;
+}
+
+.capitalCard p {
+  margin: 0%;
   text-align: center;
-  padding: 60px;
-  box-shadow: 2px 2px 16px 1px rgba(0, 0, 0, 0.3);
+  font-size: 20px;
+}
+
+.capitalCont {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  padding: 1em 0;
 }
 
+.apCont .co h1 {
+  margin: 0%;
+  border-bottom: 3px solid rgb(20, 116, 194);
+  border-radius: 2rem;
+  width: 50%;
+}
+
+.apCont .co p {
+  border: 1px solid rgb(34, 128, 216);
+  border-radius: 1rem;
+  padding: 0.5em;
+  margin: 0.25em;
+  font-size: 1.25em;
+  width: 100%;
+  box-shadow: 2px 2px 16px 1px rgba(32, 78, 165, 0.3);
+}
+
+.apCont .co {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  width: 100%;
+  align-items: center;
+  font-size: 20px;
+}
+
+.filterBox {
+  padding: 0.3em;
+  font-size: 1em;
+  width: 100%;
+  background-color: white;
+}
```

Comment:
- [ ] Kindly be noticed that you should add `color: black;` to the fillerBox class to make the text shown on the input.

Before | After
--------- | -------
![Screenshot from 2022-08-02 01-46-52](https://user-images.githubusercontent.com/33752735/182263333-41891637-e9fb-43f1-9fb8-b3c9a4a4f7cd.png) | ![Screenshot from 2022-08-02 01-46-34](https://user-images.githubusercontent.com/33752735/182263342-72ebe045-7b3f-4c8f-97cb-8dfcca4d9836.png)